onelogin_aws_cli/configuration.py

```diff
@@ -7,10 +7,12 @@
 class ConfigurationFile(configparser.ConfigParser):
     """Represents a configuration ini file on disk"""
 
+    DEFAULTS = dict(save_password=False)
+
     def __init__(self, config_file=None):
         super().__init__(default_section='defaults')
 
-        self['defaults'] = dict(save_password=False)
+        self[self.default_section] = self.DEFAULTS
```

Comment:
The reason I added the dict is to create a copy (in case changes in to the defaults across multiple instantiations in the unittests cross-pollute)

Reply:
cool, I rolled that commit back

Reply:
I have to come back to the test case another day, so we can either wait, or I'll create an issue to add the new test case.

Reply:
We can just push this and worry about it in one of the other PRs =).